src/services/authService.js

```diff
@@ -0,0 +1,32 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import Helper from '../utils/helpers';
+import { User } from '../models';
+
+const jwtSecret = process.env.JWTSECRET;
+
+/**
+ * Class to authenticate User
+ */
+export default class AuthService {
+/**
+ * @param {string} email - valid email address
+ * @param {string} password - valid password
+ * @return {string} - token
+ */
+  static async login(email, password) {
+    const result = await User.findOne({ where: { email, is_verified: true } });
+
+    if (!result) throw new Error('Invalid Credentials');
```

Comment:
A user login details (email and password) can be valid but not verified `is_verified : false`. I don't think `'Invalid Credentials'` properly is a descriptive message for that. The user might not know what to do next

Reply:
is_verified is set to true if the user successfully verify his or her account via email link sent to them